admin_app_2/src/components/ContentModal.tsx

```diff
@@ -0,0 +1,166 @@
+          }}
+        >
+          <Layout.FlexBox
+            flexDirection={"row"}
+            {...appStyles.justifyContentSpaceBetween}
+          >
+            <Typography variant="h5">Content #{content_id}</Typography>
+            <Close onClick={onClose} />
+          </Layout.FlexBox>
+
+          <Layout.Spacer multiplier={1} />
+
+          <Layout.FlexBox
+            flexDirection={"column"}
+            sx={{
+              height: "80%",
+            }}
+          >
+            <LanguageButtonBar expandable={false} />
+            <Layout.FlexBox
+              flex={1}
+              flexDirection={"column"}
+              sx={{
+                p: sizes.baseGap,
+                mr: sizes.baseGap,
+                overflowY: "auto",
+                border: 1,
+                borderColor: appColors.outline,
+                borderRadius: 1,
+              }}
+            >
+              <Layout.Spacer multiplier={1} />
+              <Typography variant="subtitle1">{title}</Typography>
+              <Layout.Spacer multiplier={1} />
+              <Typography variant="body2">{text}</Typography>
+            </Layout.FlexBox>
+            <Layout.Spacer multiplier={1} />
+
+            <Layout.FlexBox
+              flexDirection={"row"}
+              gap={sizes.smallGap}
+              {...appStyles.justifyContentSpaceBetween}
+            >
+              <Layout.FlexBox
+                {...appStyles.alignItemsCenter}
+                flexDirection={"row"}
+              >
+                <Link href={`/content/edit?content_id=${content_id}`}>
+                  <Button variant="contained" color="primary">
+                    <Edit />
+                    <Layout.Spacer horizontal multiplier={0.4} />
+                    Edit
+                  </Button>
+                </Link>
+                <Layout.Spacer horizontal multiplier={1} />
+                <Delete
+                  style={{ cursor: "pointer" }}
+                  color="primary"
+                  fontSize="small"
+                  onClick={() => {
+                    if (
+                      window.confirm(
+                        "Are you sure you want to delete this content?",
+                      )
+                    ) {
+                      apiCalls
+                        .deleteContent(content_id)
+                        .then(() => {
+                          onClose();
+                        })
+                        .then(() => {
+                          onDelete();
+                        });
+                    }
+                  }}
+                />
+              </Layout.FlexBox>
+              <Layout.FlexBox
+                {...appStyles.alignItemsCenter}
+                flexDirection={"row"}
+              >
+                <Typography variant="body2" color={appColors.darkGrey}>
+                  Last modified on{" "}
+                  {new Date(last_modified).toLocaleString("en-UK", {
```

Comment:
Changing to undefined means it uses the user's locale instead of imposing the UK format. Also, it seems that the time zone is actually our backend's timezone and not the user's timezone - adding a backlog ticket for this.

```suggestion
                  {new Date(last_modified).toLocaleString(undefined, {
```